### DISCUSSION TITLE
Hot Water

### DISCUSSION THREAD

**Discussion by jpmorley** [binsentsu/home-assistant-solaredge-modbus#141] [Q&A]
I had my SolarEdge Smart Hot water controller installed recently and I'm a bit underwhelmed by the way the app and monitoring reports it's usage - I was hoping for a bit more in the way of stats.  Seems to me it's just treated as consumption and apart from the time it's being used, you don't know what's actually happened during that day.
I have my modbus setup and linked into Home Assistant, does anyone know if any additional entities are available related to the Hot water controller? None have been automatically added yet, but I expect I need to delete the integration and reload it to add them, which I'm not keen on doing unless it's worth my while....

**Comment by ChristophCaina** [comment]
Hi, as far as I know, there are currently no implementations for such "additional" hardware implemented.
If there's a documentation about the Modbus addresses and registers, it could be possible to add them... but honestly speaking ... I don't know how active the development on such features and additions might be...

**Reply by Homebrew09** [reply]
I had the same issue.
If anybody is still looking for a solution I just found this HACS Integration. Works really nice for me.

https://github.com/barnold89/solaredge_hotwater
